concept/type/entitytype.feature

```diff
@@ -748,9 +913,12 @@ Feature: Concept Entity Type
     When put attribute type: email, with value type: string
     When put entity type: person
     When entity(person) set owns attribute type: email
+    Then entity(person) get owns overridden attribute(email) is null: true
     When put entity type: customer
     When entity(customer) set supertype: person
     Then entity(customer) set owns key type: email
+    Then entity(customer) get owns overridden attribute(email) is null: false
+    Then entity(customer) get owns overridden attribute(email) get label: email
```

Comment:
This isn't as meaningless a check as it might seem at the first glance, illustrated by the checks on lines 916 and 920.